astropy/io/votable/tree.py

```diff
@@ -25,7 +25,7 @@
 from ...utils.xml.writer import XMLWriter
 from ...utils.exceptions import AstropyDeprecationWarning
 from ...utils.misc import InheritDocstrings
-
+from ...units import Quantity
```

Comment:
Quantity is not actually needed in this file. I was earlier checking for Quantity objects in this file so that I could use .info for attributes, but that was removed in the previous commit but I forgot to remove this. I will remove this.